### ISSUE TITLE
Handle node API errors

### ISSUE THREAD

**Issue by paulrberg** [MainframeHQ/hifi-liquidator#11]
Whenever Infura returns the following error (and presumably others like it):

```
Error: Deserialization Error: expected value at line 1 column 1. Response:
<html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
</body>
</html>
```

The liquidator program exits, which is not good.

**Comment by paulrberg** [comment]
Just happened again:

```
thread 'tokio-runtime-worker' panicked at 'WS Server panic: NoResponse', /home/ec2-user/.cargo/git/checkouts/ethers-rs-c3a7c0a0ae0fe6be/fc4b3e4/ethers-providers/src/transports/ws.rs:189:38
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Error: oneshot canceled
```

**Comment by paulrberg** [comment]
Expecting the Infura API to have 0% downtime or errors is unpractical. The real solution to this problem is Kubernetes, which we implemented in [007c2d3](https://github.com/MainframeHQ/hifi-liquidator/commit/007c2d3255c5067788da93dde5c915820b001a3c).
